### PR TITLE
Fix an issue in RequestParser when the query string is null

### PR DESCRIPTION
**Pull request by teohhanhui**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | N/A

`Request::getQueryString` may return `null`.

### CODE REVIEW

**Comment by teohhanhui** [comment]
ping @api-platform/core-team

File: src/Util/RequestParser.php [hunk 1]
```diff
@@ -32,7 +32,7 @@
      */
     public static function parseAndDuplicateRequest(Request $request) : Request
     {
-        $query = self::parseRequestParams($request->getQueryString());
+        $query = self::parseRequestParams($request->getQueryString() ?? '');
```
Comment (vincentchalamon) [inline src/Util/RequestParser.php]: Isn't it supposed to be `?:` instead of `??`?

Comment (teohhanhui) [inline src/Util/RequestParser.php]: `??` is the null coalesce operator. `?:` (shorthand version of ternary operator) works here too.

Comment (vincentchalamon) [inline src/Util/RequestParser.php]: :+1:

**Comment by vincentchalamon** [comment]
LGTM

**Comment by theofidry** [comment]
:+1:

**Pull request merged**